### PR TITLE
[css-text-3] TC for making sure break-word also breaks &nbsp;

### PR DESCRIPTION
**Pull request by frivoal**
To be merged if we resolve in favor of https://lists.w3.org/Archives/Public/www-style/2015Jun/0100.html
(and if the test looks sane, of course).

### CODE REVIEW

**Comment by kojiishi** [comment]
The test looks good to me. I'm not very familiar with format guidelines, but I think:

* [Use spaces rather than tabs for indentation](http://testthewebforward.org/docs/test-format-guidelines.html#style-rules)
* IIUC, the [specification links](http://testthewebforward.org/docs/css-metadata.html#specification-links) should point to TR rather than ED, but can't find references. Maybe I'm wrong, not sure.

**Comment by frivoal** [comment]
> The test looks good to me. I'm not very familiar with format guidelines, but I think:
> 
> Use spaces rather than tabs for indentation http://testthewebforward.org/docs/test-format-guidelines.html#style-rules

Fixed

> IIUC, the specification links http://testthewebforward.org/docs/css-metadata.html#specification-links should point to TR rather than ED, but can't find references. Maybe I'm wrong, not sure.
> My understanding is that it is OK to point to an ED when the TR draft does not contain the text you want to refer to. The automated validation checks did come out clean.

In this case the TR draft is ambiguous, and may or may not support the test assertion, depending on how you read it, but the ED is clear.

So I picked the ED. If you still want me to link to TR, I'll change that.
- Florian

File: css-text-3/reftest/overflow-wrap-break-word-001-ref.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Reference File</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<style>
+div {
+	position: relative;
```
Comment (kojiishi) [inline css-text-3/reftest/overflow-wrap-break-word-001-ref.html]: Can fix this tab too?

Comment (frivoal) [inline css-text-3/reftest/overflow-wrap-break-word-001-ref.html]: > In css-text-3/reftest/overflow-wrap-break-word-001-ref.html <https://github.com/w3c/csswg-test/pull/780#discussion_r32126045>:
> > @@ -0,0 +1,16 @@
> > +<!DOCTYPE html>
> > +<meta charset="utf-8">
> > +<title>CSS Text Reference File</title>
> > +<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
> > +<style>
> > +div {
> > +	position: relative;
> Can fix this tab too?
> 
Oops, sorry I missed it earlier. Fixed now.

 - Florian

**Pull request merged**